integrations/google-analytics/lib/index.js

```diff
@@ -774,7 +774,10 @@ GA.prototype.checkoutStepCompletedEnhanced = function(track) {
     option: options
   });
 
-  window.ga(self._trackerName + 'send', 'event', 'Checkout', 'Option');
+  window.ga(self._trackerName + 'send', 'event', 'Checkout', 'Option', {
+    step: props.step || 1,
+    option: options
+  });
```

Comment:
LGTM! Do we know if there is any negative side effects of sending the step and option twice here? I don't see any based off your tests but was just wondering if you noticed anything in your local testing?

Reply:
Negative, not that I've noticed in my end-to-end tests. I have a customer testing this first so I'll give them some extra time to look at reports and confirm.